doc/refcard/refcard.tex

```diff
@@ -237,44 +240,47 @@
     \texttt{r}   & Evaluate current \textbf{r}egion \\
     \texttt{q}   & \textbf{Q}uit window \\
     \texttt{k}   & \textbf{K}ill this buffer\\
-    \texttt{x}   & Kill this buffer and return (`e\textbf{x}it)\\
+    \texttt{x}   & Kill this buffer and return to ESS\\
     \texttt{h}   &\textbf{H}elp on another object \rule{0pt}{3ex}\\
     \texttt{?}   & Help for this mode \\
     \texttt{a}   & Display \textbf{a}propos\\
     \texttt{i}   & Display \textbf{i}ndex\\
     \texttt{v}   & Display \textbf{v}ignettes\\
     \texttt{w}   & Display this help in \textbf{w}eb bro\textbf{w}ser\\
+    \texttt{/}   & Search (also on \texttt{C-s})\\
   \end{tabI}\\[0.5cm]
 
   \Sect{ESS tracebug}
   %% ~~~~~~~~~~~~~~~~
-  Commands in \textbf{\underline{ess-dev-map}} (\textbf{C-c C-t}):
+  Commands in \texttt{ess-dev-map} (\texttt{C-c C-t}):
 
   \begin{tabular}{p{20mm}l}
     \texttt{?} & Show key help \\
-    \texttt{C-b, b}& Set BP (repeat to cycle)\\
+    \texttt{b}& Set BP (repeat to cycle)\\
+    \texttt{C-b, B}& Set conditional BP\\
     \texttt{C-k, k}& Kill BP\\
+    \texttt{C-K, K}& Kill all BPs\\
     \texttt{C-n, n}& Goto next BP\\
     \texttt{C-p, p}& Goto previous BP\\
-    \texttt{`} &Show R Traceback (also on \texttt{C-c `}\\
-    \texttt{\~} &Show R call stack (also on \texttt{C-c ~}\\
+    \texttt{\textasciigrave} & Show R traceback (also on \texttt{C-c \textasciigrave})\\
+    \texttt{\textasciitilde} & Show R call stack (also on \texttt{C-c \textasciitilde})\\
     \texttt{C-e, e}& Toggle error action (cycle)\\
     \texttt{C-d, d}& Flag for debugging\\
     \texttt{C-u, u}& Un-flag debugged objects\\
     \texttt{C-w, w} & Watch window\\
-    \texttt{0..9, q}& Recover commands\\
+    %\texttt{0..9, q}& Recover commands\\ % seems to be obsolete
   \end{tabular}
 
-  Commands in \textbf{\underline{ess-debug-mode-map}}\\
+  Commands in \texttt{ess-debug-mode-map}\\
   (active during debugging):
 
   \begin{tabular}{p{20mm}l}
     \texttt{M-C}&  Continue \\
     \texttt{M-N}&  Next line\\
     \texttt{M-Q}&  Quit\\
     \texttt{M-U}&  Up frame\\
-    \texttt{C-M-S-c }&  Continue Multiple\\
-    \texttt{C-M-S-n }&  Next Multiple\\
+    \texttt{M-C-C}&  Continue Multiple\\
```

Comment:
I think the previous version is more explicit, and thus better. The M-C-C is a bit confusing.

Reply:
The previous version has been restored.